api/certs.yml

```diff
@@ -22,29 +22,64 @@ paths:
           description: Failed due to malformed JSON.
         '500':
           description: Unexpected server-side error ocurred.
-  /certs/{thingId}:
+  /certs/{thingID}/list:
```

Comment:
Shouold here `thingID` be part of the URL, or rather a param? @nmarcetic @dusanb94 opinion?